package.json

```diff
@@ -17,7 +17,8 @@
     "type-check": "tsc --noEmit",
     "test": "jest",
     "test:ci": "yarn lint && yarn type-check && yarn test",
-    "prepublishOnly": "tsc --declaration"
+    "build": "tsc -p tsconfig.dist.json --declaration",
```

Comment:
Made this a separate command because I found `yarn build --watch` to be helpful when working on the cli.